src/org/labkey/targetedms/view/pharmacokinetics.jsp

```diff
@@ -52,7 +52,7 @@
     %>
     <labkey:panel title="<%=subgroupTitle%>" type="portal">
         <div id="targetedms-fom-export" class="export-icon" data-toggle="tooltip" title="Export to Excel">
-            <i class="fa fa-file-excel-o" onclick="exportExcel('<%=h(subgroup)%>')"></i>
+            <%=iconLink("fa fa-file-excel-o", "", null).onClick("exportExcel('" + h(subgroup) + "')") %>
```

Comment:
I suspect we don't need the `h()` call here. I hope `onClick()` already does the appropriate encoding for us.